Fixed POS/POW tables.

KaneoHunter opened the pull request:
Corrected circulation info.
Corrected a BR on the POS table.
Added a note about seesaw to the POS table.
Replaced the seesaw column with circulation on the POS table
Changed up the spacing to make the table look cleaner

KaneoHunter committed:
Merge pull request #1 from bulwark-crypto/master

Updating to latest.

KaneoHunter committed:
Bulwark PoW and PoS block reward table: whitepaper of the year edition.

"The definitive Bulwark BR table experience. 10/10." - TotalBiscuit

KaneoHunter committed:
Bulwark PoW and PoS block reward table: whitepaper of the year edition.

"The definitive Bulwark BR table experience. 10/10." - TotalBiscuit

KaneoHunter committed:
Merge remote-tracking branch 'origin/master'

padraiq commented on source/10_chapter_5.md:
```diff
@@ -26,47 +26,51 @@ Subsidy		 Block 	    	 PoW	  MN     Circulation
+1.6250    1728001+           10%         ��                   In perpetuity
+=======
+1.6250    1728000+           10%         ��                   In perpetuity
+>>>>>>> master
```
Do not leave remnants of merge conflicts in PR

KaneoHunter committed:
Amended numbers

Amended numbers to more accurately reflect slowstart, also removed merge conflict stuff.

KaneoHunter commented:
Amended numbers to reflect slowstart at Levi's request. Also removed remnants of the merge conflict.
